modules/docs/arrow-docs/docs/docs/patterns/glossary/README.md

```diff
@@ -24,6 +24,21 @@ branching in code with [`Either`]({{ '/docs/datatypes/either' | relative_url }})
 catching exceptions with [`Try`]({{ '/docs/datatypes/try' | relative_url }}),
 or interacting with the platform the program runs in using [`IO`]({{ '/docs/effects/io' | relative_url }}).
 
+The name datatype comes from the representation of these patterns using plain data, with `data` and `sealed` classes.
```

Comment:
Is it like that? I thought it was more related to the data types being "the data" of a functional program (as in they're used to hold the data to operate on with typeclasses combinators which are the connectors)

Reply:
We use the term `Data type` to refer to any data structure for which a type class instance can be targeted so it covers much more encodings than the traditional Algebraic data type encoding,in kotlin based in ADTs. @pakoito in your change you are referring exclusively to Algebraic data types but other data types for example church encoded data types we call them also `data types`. Ex `List` in the Kotlin std lib